xlml/utils/xpk.py

```diff
@@ -60,14 +61,18 @@ def run_workload(
   """Run workload through xpk tool."""
 
   with tempfile.TemporaryDirectory() as tmpdir:
+    if accelerator_type == GpuVersion.XPK_H100.value:
+      multi_keyword = "num-nodes"
+    else:
+      multi_keyword = "num-slices"
```

Comment:
@Obliviour Do you think we could consolidate the two flags upstream in xpk? Similar to how we use `device-type` for both TPU and GPU.